invoices_app/__init__.py

```diff
@@ -1,3 +1,7 @@
+MAX_SIZE_FILE = 5242880
+
+ALLOWED_FILE_EXTENSIONS = ['.pdf']
```

Comment:
This limitation only applies to `invoices_app` or all apps?

Reply:
It only applies to the files of invoices_app

Reply:
In that case, could we rename this to 

> INVOICE_MAX_SIZE_FILE

 and 

> INVOICE_ ALLOWED_FILE_EXTENSIONS

Reply:
resolved